### PR TITLE
Add promotion section

### PR DESCRIPTION
**Pull request by IsmaelVillavicencio**
Link : https://github.com/magma-labs/shopify-themes/issues/27
Originals
desktop 1280
mobile portrait

how to looks
desktop
mobil

### CODE REVIEW

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 1]
```diff
@@ -31,17 +31,75 @@ p{
...
+}
+.title-icon {
+  display: block;
+  overflow: hidden;
+  position: absolute;
+  top: 25px;
+  width: 100%;
+}
+
+.title-icon:before, .title-icon:after {
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 2]
```diff
@@ -31,17 +31,75 @@ p{
...
+  display: inline-block;
+  vertical-align: middle;
+  position: relative;
+  width: 100%;
+  border-top-style: solid;
+  border-top-width: 1px;
+  border-color: #E83671; 
 }
 
+.title-icon:before {
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 3]
```diff
@@ -31,17 +31,75 @@ p{
...
+  right: 5px;
+  margin-left: -80%;
+}
+.title-icon:after {
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 4]
```diff
@@ -31,17 +31,75 @@ p{
...
+  left: 5px;
+  margin-right: -50%;
+}
+
+.title-icon-img {
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 5]
```diff
@@ -31,17 +31,75 @@ p{
...
+  width: 40px;
+  vertical-align: middle;
+}
+
+.promotion{
+  padding: 10px;
+}
+
+.box-promotions{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 1. Please reorder the properties in alphabetical order
2. Missing Space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 6]
```diff
@@ -31,17 +31,75 @@ p{
...
+  position: relative;
+  height: auto;
+  background: #F3F9EF;
+  background-image: url('../images/home/banner_image_home_tablet@2x.jpg');
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: right;
+}
 
+.promotion-message{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 7]
```diff
@@ -201,9 +259,8 @@ p{
 }
 
 @media only screen and ( max-width: 768px ) {
-  /* footer */
 
-    .btn-slider, .btn-call-action{
+  .btn-slider, .btn-call-action{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

> Boy Scout Rule

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 8]
```diff
@@ -245,6 +302,19 @@ p{
     display: none;
   }
 
+  .box-promotions{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 9]
```diff
@@ -659,9 +730,34 @@ p{
     text-align: center;
   }
 
+  .title-icon{
+    top: 40px;
+  }
+
+  .promotion-message{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 10]
```diff
@@ -31,17 +31,75 @@ p{
   padding: 0;
   width: auto;
 }
+/* ------ promotion ------ */
+.p-prom-description, .p-prom-price{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: missing space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 11]
```diff
@@ -31,17 +31,75 @@ p{
...
+  margin-top: 0px;
+  margin-bottom: 0px;
+  text-align: left;
+  letter-spacing: 3px;
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 12]
```diff
@@ -31,17 +31,75 @@ p{
...
-  margin:0;
-  padding: 0;
-  width: auto;
+.p-prom-price{
+  color: #E83671;
+}
+.prom-price{
+  font-family: 'MerriweatherSans-Bold';
+}
+.title-icon {
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 13]
```diff
@@ -31,17 +31,75 @@ p{
...
-  align-items: center;
-  background-color: #F3F9EF;
-  height: 300px;
-  margin:0;
-  padding: 0;
-  width: auto;
+.p-prom-price{
+  color: #E83671;
+}
+.prom-price{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: missing space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 14]
```diff
@@ -31,17 +31,75 @@ p{
...
+}
 
-.promotion {
-  align-items: center;
-  background-color: #F3F9EF;
-  height: 300px;
-  margin:0;
-  padding: 0;
-  width: auto;
+.p-prom-price{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: missing space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 15]
```diff
@@ -31,17 +31,75 @@ p{
...
+  left: 5px;
+  margin-right: -50%;
+}
+
+.title-icon-img {
+  width: 40px;
+  vertical-align: middle;
+}
+
+.promotion{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: missing space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 16]
```diff
@@ -659,9 +730,34 @@ p{
...
+    top: 40px;
+  }
+
+  .promotion-message{
+    left: 40px;
+    bottom: 40px;
+    font-size: 40px;
+  }
+
+  .box-promotions{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 1. missing space before `{`
2. Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 17]
```diff
@@ -659,9 +730,34 @@ p{
...
+    height: 400px;
+    background-image: url('../images/home/banner_image_home_tablet@2x.jpg');
+  }
+
+  .p-prom-description{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 1. missing space before `{`
2. Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 18]
```diff
@@ -659,9 +730,34 @@ p{
...
+    width: 500px;
+    font-size: 50px;
+  }
+
+  .p-prom-price{
+    width: 450px;
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 1. missing space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 19]
```diff
@@ -717,4 +813,28 @@ p{
   .social-media{
     border: 0px;
   }
+
+  .title-icon{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 1. missing space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 20]
```diff
@@ -717,4 +813,28 @@ p{
...
+    top: 50px;
+  }
+
+  .promotion-message{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 1. missing space before `{`
2. Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 21]
```diff
@@ -717,4 +813,28 @@ p{
...
+    left: 60px;
+    bottom: 35px;
+    font-size: 50px;
+  }
+
+  .box-promotions{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 1. missing space before `{`
2. Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 22]
```diff
@@ -717,4 +813,28 @@ p{
...
+    height: 500px;
+    background-image: url('../images/home/banner_image_home_1024@2x.jpg');
+  }
+
+  .p-prom-description{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 1. missing space before `{`
2. Please reorder the properties in alphabetical order

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 23]
```diff
@@ -717,4 +813,28 @@ p{
...
+    width: 500px;
+    font-size: 60px;
+  }
+
+  .p-prom-price{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: 1. missing space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css (hunk 10 above)
Comment (JuanCrg90) [reply MH-FiorellaTheme5/source/stylesheets/styles.css]: ☝️

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 24]
```diff
@@ -625,30 +696,30 @@ p{
...
     right: 15px;
-    bottom: 110px;
-    font-size: 20px;
+    text-align: right;
   }
 
   .btn-slider{
+    bottom: 50px;
+    font-size: 24px;
+    padding: 10 60px;
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: why you don't specify the unit for the first value?

File: MH-FiorellaTheme5/source/stylesheets/styles.css (hunk 9 above)
Comment (JuanCrg90) [reply MH-FiorellaTheme5/source/stylesheets/styles.css]: ?

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 25]
```diff
@@ -690,31 +786,55 @@ p{
...
-    text-align: right;
+    font-size: 32px;
     position: absolute;
     right: 15px;
-    bottom: 110px;
-    font-size: 32px;
+    text-align: right;
   }
 
   .btn-slider{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: please add the missing space before `{`

> Boy Scout rule

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 26]
```diff
@@ -690,31 +786,55 @@ p{
...
+    font-size: 50px;
+    left: 60px;
+  }
+
+  .box-promotions{
+    background-image: url('../images/home/banner_image_home_1024@2x.jpg');
+    height: 500px;
+  }
+
+  .p-prom-description{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please add the missing space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 27]
```diff
@@ -690,31 +786,55 @@ p{
...
+    font-size: 60px;
+    width: 500px;
+  }
+
+  .p-prom-price{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Please add the missing space before `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 28]
```diff
@@ -690,31 +786,55 @@ p{
...
     position: absolute;
-    width: 300px;
     right: 15px;
-    padding: 10 60px;
-    font-size: 24px;
-    bottom: 50px;
+    width: 300px;
   }
 
   .box-footer{
```
Comment (JuanCrg90) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: please add the missing space before `{`

> Boy Scout Rule

File: MH-FiorellaTheme5/source/partials/_promotion.erb [hunk 29]
```diff
@@ -1,3 +1,11 @@
 <section class="promotion">
-  <p class="text">Promotion</p>
+	<div class="box-promotions">
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/partials/_promotion.erb]: Your indentation is kind of weird here, can you fix it please.

File: MH-FiorellaTheme5/source/partials/_promotion.erb [hunk 30]
```diff
@@ -1,3 +1,11 @@
...
+		<div class="title-icon">
+			<img class="title-icon-img" src="./images/home/image.png" alt="">
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/partials/_promotion.erb]: You don't need the `.` before `images`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 31]
```diff
@@ -31,54 +31,113 @@ p{
...
-  height: 300px;
-  margin:0;
-  padding: 0;
-  width: auto;
+  padding: 10px;
 }
 
+.box-promotions {
+  background: #F3F9EF;
+  background-image: url('../images/home/banner_image_home_tablet@2x.jpg');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: no need to use the `..`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 32]
```diff
@@ -89,71 +148,70 @@ p{
...
 
 .ul-footer li a {
-  font-family: "MerriweatherSans-Regular";
-  text-decoration:none;
+  color: #000;
   display: block;
+  font-family: "MerriweatherSans-Regular";
   margin: 10px;
-  color: #000;
+  text-decoration:none;
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Missing a space after the `:` (colon).

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 33]
```diff
@@ -162,89 +220,101 @@ p{
   margin: 10px;
 }
 
-.icon_facebook, .icon_pinterest, .icon_gplus{
-  width: 40px;
+.icon_facebook, .icon_gplus, .icon_pinterest{
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: space after the `pinterest`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 34]
```diff
@@ -162,89 +220,101 @@ p{
...
   height: 40px;
+  width: 40px;
 }
 
-.icon_facebook{
+.icon_facebook {
   background-image: url('../images/home/icon_facebook.png');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: remove the `..` before `images`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 35]
```diff
@@ -162,89 +220,101 @@ p{
...
-  background-size: 40px;
   background-repeat: no-repeat;
+  background-size: 40px;
 }
 
-.icon_pinterest{
+.icon_pinterest {
   background-image: url('../images/home/icon_pinterest.png');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: And here

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 36]
```diff
@@ -162,89 +220,101 @@ p{
...
-  background-size: 40px;
   background-repeat: no-repeat;
+  background-size: 40px;
 }
 
-.icon_gplus{
+.icon_gplus {
   background-image: url('../images/home/icon_gplus.png');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Here

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 37]
```diff
@@ -162,89 +220,101 @@ p{
...
-  background-size: 40px;
   background-repeat: no-repeat;
+  background-size: 40px;
 }
 
-.icon_facebook:hover{
+.icon_facebook:hover {
   background-image: url('../images/home/bicon_facebook.png');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: here

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 38]
```diff
@@ -162,89 +220,101 @@ p{
...
   background-size: 40px;
 }
 
-.icon_pinterest:hover{
+.icon_pinterest:hover {
   background-image: url('../images/home/bicon_pinterest.png');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: here

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 39]
```diff
@@ -162,89 +220,101 @@ p{
...
   background-size: 40px;
 }
 
-.icon_gplus:hover{
+.icon_gplus:hover {
   background-image: url('../images/home/bicon_gplus.png');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: here

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 40]
```diff
@@ -162,89 +220,101 @@ p{
...
-    bottom: 50px;
   }
 
-  .footer_fiorella, .footer_info, .footer_contact{
+   .footer_contact, .footer_fiorella, .footer_info {
     display: none;
   }
 
+  .box-promotions {
+    background-image: url('../images/home/banner_image_home_mobil@2x.jpg');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: and here

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 41]
```diff
@@ -598,81 +668,107 @@ p{
...
+    padding: 10px 60px;
     position: absolute;
     right: 15px;
-    padding: 10 60px;
-    font-size: 24px;
-    bottom: 50px;
   }
 
-  .footer_fiorella, .footer_info, .footer_contact{
+  .footer_contact, .footer_fiorella, .footer_info{
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: space before the bracket `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 42]
```diff
@@ -598,81 +668,107 @@ p{
...
+  }
+
+  .promotion-message {
+    bottom: 40px;
+    font-size: 40px;
+    left: 40px;
+  }
+
+  .box-promotions {
+    background-image: url('../images/home/banner_image_home_tablet@2x.jpg');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: remove dots

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 43]
```diff
@@ -598,81 +668,107 @@ p{
...
+  }
+
 }
 
 @media only screen and ( min-width: 1024px ){
-  .btn-slider, .btn-call-action{
-    width: 325px;
-    height: 73px;
+
+  .btn-call-action, .btn-slider{
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: space before bracket `{`

File: MH-FiorellaTheme5/source/stylesheets/styles.css [hunk 44]
```diff
@@ -684,37 +780,60 @@ p{
...
+  }
+
+  .promotion-message {
+    bottom: 35px;
+    font-size: 50px;
+    left: 60px;
+  }
+
+  .box-promotions {
+    background-image: url('../images/home/banner_image_home_1024@2x.jpg');
```
Comment (albertm805) [inline MH-FiorellaTheme5/source/stylesheets/styles.css]: Remove `..` before `images`

**Pull request merged**